validate/setup-guides/aws/createEC2.md

```diff
@@ -63,110 +75,52 @@ The term *new* is used simply because (at the time of this documentation creatio
-:::
+### Name your instance
 
+![](/img/validator_nodes/node-aws-ec2-name-tag.png)
 :::tip
 It is **highly** recommended (but not required) to use a `Debian` distribution.
 :::
 
-![](/img/validator_nodes/node-aws-ec2-6.png)
-
-#### INSTANCE TYPE
-
-Click on **All instance families** to drop down the list
-
-![](/img/validator_nodes/node-aws-ec2-7.png)
-
-We will choose `a1`. Constellation Network wants us to use `a1.xlarge`
-
-Click on **`a1.xlarge`**.
-
-![](/img/validator_nodes/node-aws-ec2-8.png)
-
-Click on Next: `Configure instance Details`.
-
-
-![](/img/validator_nodes/node-aws-ec2-9.png)
-
-#### EC2 Details
-
-Need to supply AWS some details
-
-Switch our Auto-assign Public IP to Disabled. We are going to create an EIP later in this process.
+### Pick EC2 Image 
 
-![](/img/validator_nodes/node-aws-ec2-10.png)
-
-#### STORAGE
-
-Click on Next: `Add Storage`.
-
-![](/img/validator_nodes/node-aws-ec2-11.png)
-
-Change the storage size to **`160`**.
-
-![](/img/validator_nodes/node-aws-ec2-12.png)
-
-
-The next 2 steps are **Add Tags** and **Configure Security Groups**; however, *we will skip these for now*, and come back to configure them later on in our tutorial steps.
-
-Let's get our instance up and running. Select **Review and Launch**!
-
-![](/img/validator_nodes/node-aws-ec2-13.png)
-
-We will continue configuration while it is launching
-
-:::danger Possible Warning
-
-We may see these warnings, that our Node is vulnerable. We can safely ignore these because our Node will not have an IP address and can not access the Internet, with its current configuration.
-
-We will only be able to work with this EC2 instance from our dashboard only. Do not worry though, when we are done, our communications will be fixed; as well as, our security setup.
-
-![](/img/validator_nodes/node-aws-ec2-14.png)
-
-This is OK, we will fix later! 
+:::note
+You can use any Debian distribution you would like, it does not need to be Ubuntu. We will be using **`Ubuntu`** throughout our tutorial(s), so if you choose a different distribution, you may need to change some commands to match your chosen distribution.
 :::
 
-You can **launch**!
-
-![](/img/validator_nodes/node-aws-ec2-15.png)
-
-Once we hit the `Launch` we will be prompted to add our `SSH Key Pair`. You should have created one from the last section. 
+![](/img/validator_nodes/node-aws-ec2-distro.png)
 
-You will choose those keys on the next screen.
+We choose:
+  - Ubuntu
+  - 64-bit (x86)
+  - 22.04 LTS
 
-##### ASSIGN KEY PAIR
+You may choose whatever distribution (distro) and version of your chosen distro that you are most comfortable with.  **HOWEVER**, you must use a **Debian** Linux distro to participate in Constellation Network's MainNet 2.0.
 
-If you did not create an SSH key in the last section, you will have the option to create a new key here. This is **NOT** recommended.
+Most recommended are:
+  - Debian  
+  - Ubuntu
```

Comment:
I feel like this info can be removed. The debian requirement has been noted several times already. If the user follows this guide they'll choose Ubuntu and be fine.